security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,29 @@
+package crypter
+
+import "errors"
+
+const (
+	// GcmTagSize is the tag size - the difference in length between plaintext
+	// and ciphertext. From crypto/cipher/gcm.go in the Go crypto library.
+	GcmTagSize = 16
+	// NonceSize is the size of the nonce in number of bytes.
```

Comment:
Can you mention that this is the nonce size for AES-128-GCM-SHA256, AES-256-GCM-SHA384, and CHACHA20-POLY1305-SHA256? (It might not be the nonce size for a ciphersuite that we add in the future.)